notready/MNL000555.yaml

```diff
@@ -1,32 +1,58 @@
 bioguide: MNL000555
 contact_form:
-  method: post
-  action:
+  driver: chrome
   steps:
-    - visit: "http://www.mnsenaterepublicans.com/our-senators/bill-weber/"
+    - visit: https://www.senate.mn/members/email-form/1199
```

Comment:
Similar question, though in this case I don't think I see a yaml for Bill Weber's new id.